definitions/npm/ramda_v0.21.x/flow_>=v0.23.x_<=v0.27.x/ramda_v0.21.x.js

```diff
@@ -1,642 +0,0 @@
-/* @flow */
-/* eslint-disable no-unused-vars, no-redeclare */
-
-type Transformer<A,B> = {
```

Comment:
> To be honest, ramda actually gains a lot from union types rewrite. To the point that there is no use for some typings with older versions.

It's certainly true that the unions fixes make things a lot better -- but given that at least some people found this libdef useful pre-v0.28, wouldn't you agree that having a best-possible libdef for that version is better than no libdef at all? 

> The tests are not version based — my new tests won't pass with the old version

You can move test files into the `flow_>=v0.23.x_<=v0.27.x` directory and that will ensure that it only runs for that particular version of the libdef.